invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js

```diff
@@ -321,14 +325,10 @@ export const RDMEmptyResults = (props) => {
           {i18next.t("No results found!")}
         </Header>
         {queryString && (
-          <em>
-            {i18next.t("Current search")} "{queryString}"
-          </em>
+          <Button primary onClick={() => props.resetQuery()}>
+            {i18next.t("Reset query")}
```

Comment:
```suggestion
            {i18next.t("Reset search")}
```
or maybe `Start over` as in the more recent UIs.